home/templates/home/criteria.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+
+<H1>Judging Criteria for Hackaton</H1>
+<br>
```

Comment:
In general I would avoid line breaks. Could you remove them and use bootstrap grid instead?